Header header
bool wheeldropCaster
bool wheeldropLeft
bool wheeldropRight
bool bumpLeft
bool bumpRight
bool wall
bool cliffLeft
bool cliffFrontLeft
bool cliffFrontRight
bool cliffRight
bool virtualWall

